include/irr/asset/IAsset.h

```diff
@@ -42,22 +42,32 @@ class IAsset : virtual public core::IReferenceCounted
         ET_SHADER = 1u<<7u,
         //! asset::ICPUSpecializedShader
         ET_SPECIALIZED_SHADER = 1u<<8,
-        //! asset::ICPUMeshDataFormatDesc
-        ET_MESH_DATA_DESCRIPTOR = 1u<<8,
-        //! reserved, to implement later
+        //! asset::ICPURenderpassIndependentPipeline
         ET_GRAPHICS_PIPELINE = 1u<<9u,
```

Comment:
maybe try to reorganise so scene comes last, buffer view and texture before  descriptor layout,  descriptor layout before pipeline layout, and descriptor set after pipeline layout, then the pipelines

Reply:
@Crisspl reorder.

Reply:
@Crisspl TODO.

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/358#discussion_r328047980